pycqed/analysis/measurement_analysis.py

```diff
@@ -11,58 +10,46 @@
 from pycqed.analysis import analysis_toolbox as a_tools
 from pycqed.analysis import fitting_models as fit_mods
 import pycqed.measurement.hdf5_data as h5d
-from mpl_toolkits.axes_grid1 import make_axes_locatable
 from pycqed.measurement.calibration_points import CalibrationPoints
 import scipy.optimize as optimize
 import lmfit
-from collections import Counter  # used in counting string fractions
 import textwrap
 from scipy.interpolate import interp1d
 import pylab
 from pycqed.analysis.tools import data_manipulation as dm_tools
 import importlib
-import math
 from time import time
 
 try:
     import pygsti
 except ImportError as e:
     if str(e).find('pygsti') >= 0:
-        logging.warning('Could not import pygsti')
+        logger.warning('Could not import pygsti')
```

Comment:
Not a critical issue, but just to raise awareness, that the logger object is usually (in other modules of PycQED) called `log`. I know it was a problem already before.